src/ecs/resources/input/input_handler.rs

```diff
@@ -36,8 +36,8 @@ pub struct InputHandler {
     pressed_mouse_buttons: SmallVec<[MouseButton; 16]>,
     down_mouse_buttons: SmallVec<[MouseButton; 16]>,
     released_mouse_buttons: SmallVec<[MouseButton; 16]>,
-    mouse_position: Option<(i32, i32)>,
-    previous_mouse_position: Option<(i32, i32)>,
+    mouse_position: Option<(f64, f64)>,
```

Comment:
An additional note: Windows doesn't actually support this kind of precision.  Linux and Mac OS do though.